content/docs/alerting/notifications.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Notification template reference
+sort_rank: 6
+---
+# Notification Template Reference
+
+Prometheus creates and sends alerts to the Alertmanager which then sends notifications out to different receivers based on their labels.
+A receiver can be one of many integrations including: Slack, PagerDuty, email, or a custom integration via the generic webhook interface.
+
+The notifications sent to receivers are constructed via templates. The Alertmanager comes with default templates but they can also be customized.
+To avoid confusion it's important to note that the Alertmanager templates differ from [templating in Prometheus](https://prometheus.io/docs/visualization/template_reference/).
+
+The Alertmanager's notification templates are based on the [Go templating](http://golang.org/pkg/text/template) system.
+
+# Data Structures
+
+## Data
+
+`Data` is the data passed to notification templates and webhook pushes.
+
+| Name          | Type     | Notes    |
+| ------------- | ------------- | -------- |
+| Receiver | string | Defines the receiver's name that the alert will be sent to (slack, email etc.). |
+| Status | string| Defines whether or not the alert is resolved or currently firing. |
+| Alerts | [Alert](#alert) | The alert object itself ([see below](#alert)). |
+| GroupLabels | [KV](#kv) | Specifies the labels these alerts were grouped by. |
+| CommonLabels | [KV](#kv) | Specifies the labels common to all of the alerts. |
+| CommonAnnotations | [KV](#kv) | Specifies another set of common labels to all of the alerts. Used for longer additional strings of information about the alert. |
+| ExternalURL | string | Backlink to the Alertmanager that sent the alert. |
+
+## Alert
+
+`Alert` holds one alert for notification templates
+
+| Name          | Type     | Notes    |
+| ------------- | ------------- | -------- |
+| Status | string | Defines whether or not the alert is resolved or currently firing. |
+| Labels | [KV](#kv) | Specifies a set of additional labels to be attached to the alert. |
+| Annotations | [KV](#kv) | Specifies another set of labels that are not identifying for an alert instance. |
+| StartsAt | time.Time | The time the alert started firing. If omitted, the current time is assigned by the Alertmanager. |
```

Comment:
The semantics here are more complicated, I keep it vague and mention that this may be after when the alert first started firing.